lib/compilers/java.js

```diff
@@ -0,0 +1,265 @@
+// Copyright (c) 2017, Rubén Rincón
```

Comment:
I'm flattered, but the copyright is not right here :P
If you wish, you can change it to
```diff
- // Copyright (c) 2017, Rubén Rincón
+ // Copyright (c) 2019, Compiler Explorer Authors
```

Reply:
:) changed it